extensions/authentication-passport/README.md

```diff
@@ -12,6 +12,7 @@ authentication system in `@loopback/authentication@3.x`.
 
 ```sh
 npm i @loopback/authentication-passport --save
+npm i @types/passport-http --save
```

Comment:
do we want to use the following instead?
```
npm i @loopback/authentication-passport  @types/passport-http --save
```

Reply:
Actually, I broke the install into 2 different areas.

One, is to install our @authentication-passport module.

and a few paragraphs later , 2) we demonstrate how we use the adapter with 'passport-http' as an example. So this is where we remind the users that (for this example) they need to install passport-http as well as @types/passport-http